config/protocol.privnet.docker.four.yml

```diff
@@ -10,10 +10,10 @@ ProtocolConfiguration:
     - 02a7bc55fe8684e0119768d104ba30795bdcc86619e864add26156723ed185cd62
   ValidatorsCount: 4
   SeedList:
-    - node_one:20333
-    - node_two:20334
-    - node_three:20335
-    - node_four:20336
+    - http://node_one:20333
+    - http://node_two:20334
+    - http://node_three:20335
+    - http://node_four:20336
```

Comment:
It's a blatant lie, there is no HTTP there.

Reply:
Sure, fixed.